### PR TITLE
Update NuGet V3 client to 3.0.0-pre-20150220053309

### PR DESCRIPTION
**Pull request by shrimpy**
all site extension test case passed
also verified from portal, see below screenshot

image

### CODE REVIEW

File: Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs [hunk 1]
```diff
@@ -61,7 +61,7 @@ public SiteExtensionInfo(UIPackageMetadata data)
         public SiteExtensionInfo(UISearchMetadata data)
```
Comment (ahmelsayed) [inline Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs]: any reason we can't call the `SiteExtensionInfo(UIPackageMetadata)` constructor with `data.LatestPackageMetadata` ? they seem to be doing the same thing

any reason this constructor can't just be 

```
```

Comment (shrimpy) [inline Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs]: good call. only some fields need to be override. let me udpate it.

Comment (shrimpy) [inline Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs]: actually can remove it now after update search to use SearchLatestPackageResource.

**Comment by ahmelsayed** [comment]
:shipit:

**Pull request merged**